be/Cf.Application/Services/DocumentServices/ServiceDocumentService.cs

```diff
@@ -16,11 +17,26 @@ public ServiceDocumentService(Context context)
         _context = context;
     }
 
-    public async Task SavePdfAsync(byte[] pdfBytes, string? serviceId)
+    public async Task SavePdfAsync(string? base64Content, string? serviceId)
     {
+        if (string.IsNullOrEmpty(base64Content))
+        {
+            throw new BadRequestException(DomainErrors.Service.NotFound);
```

Comment:
We generally use `NotFound` when something is not found in a database in my opinion. This would be more accurate `ImagesNotProvided` or something like that.